main.js

```diff
@@ -0,0 +1,105 @@
+/* eslint no-restricted-syntax: "off", curly: "off" */
+/* eslint no-use-before-define: "off", curly: "off" */
+/* eslint no-unused-vars: "off", curly: "off" */
+/* eslint func-names: "off", curly: "off" */
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+
+  this.info = function () {
+    return (`This book is called ${this.title} from ${this.author} and it has ${this.pages} pages and${this.convert()}`);
+  };
+
+  this.convert = function () {
+    if (this.read) {
+      return ' this book has been read';
+    }
+
+    return ' this book has NOT been read';
+  };
+}
+
+const myLibrary = [];
```

Comment:
- [ ] **(OPTIONAL)** It has data persistence by saving, retrieving, and updating the library books using [the localstorage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/localstorage_stretch_requirement.png) from [Odin Project](https://www.theodinproject.com/courses/javascript/lessons/library))